app/views/layouts/darkswarm.html.haml

```diff
@@ -50,4 +33,23 @@
 
           #footer
     %loading
+
+    %script{:src => "https://js.stripe.com/v3/", :type => "text/javascript"}
+    %script{src: "//maps.googleapis.com/maps/api/js?libraries=places,geometry#{ ENV['GOOGLE_MAPS_API_KEY'] ? '&key=' + ENV['GOOGLE_MAPS_API_KEY'] : ''} "}
```

Comment:
we need to test google maps and make sure it still works - why do we include google maps in ALL frontoffice pages? :-O

Reply:
The Maps API is used on `/map`, `/shops`, `/groups`, and `/producers`. The Stripe API definitely doesn't need to be loaded on every page though!

Reply:
Stripe recommends using it on every page because it helps them collecting data and detect fraud. But honestly, I would only load it if there is something in the cart of an order cycle that uses Stripe. Should we open an issue?
https://stripe.com/docs/stripe-js/reference#including-stripejs

The documentation says nothing about where to include it. So I guess it's fine at the bottom.